app/views/layouts/_header.html.haml

```diff
@@ -0,0 +1,16 @@
+%header
+  .menu-wrapper
+    %a.logo{:href => root_path}
+      %span.logo-icon
+        Insert logo here
+
+    = link_to "Postear nuevo evento", new_event_path
```

Comment:
variables and code go in English I agree, UI on the other hand is to be used here in CR, official language here is Spanish